pkg/services/team/model.go

```diff
@@ -54,8 +54,10 @@ type DeleteTeamCommand struct {
 }
 
 type GetTeamByIDQuery struct {
```

Comment:
Maybe appropriate to change to `type GetTeamQuery`?

Similarly below, maybe it makes sense to change the method `GetTeamByID` on the service to just `GetTeam`?